zcash_client_sqlite/src/wallet.rs

```diff
@@ -1458,6 +1458,20 @@ pub(crate) fn get_transparent_balances<P: consensus::Parameters>(
     Ok(res)
 }
 
+/// Returns a vector with the IDs of all accounts known to this wallet.
+pub fn get_accounts(conn: &rusqlite::Connection) -> Result<Vec<AccountId>, SqliteClientError> {
```

Comment:
```suggestion
pub(crate) fn get_accounts(conn: &rusqlite::Connection) -> Result<Vec<AccountId>, SqliteClientError> {
```

Reply:
Done. I also fixed the name of this function to match the others (`get_account_ids`).